tests/unit/data/old/data.json

```diff
@@ -40,7 +40,10 @@
         "incorrect": "No 2",
         "correct": "Yes 2"
       },
-      "zone": "Zone 2",
+      "zones": [
+        "Zone 2",
+        "Zone 1"
+      ],
```

Comment:
@haikuginger I'm not sure if we should modify this file. There are no existing problems using the old data format that have multiple zones defined for a single item, but this change seems to suggest otherwise (so it's a little confusing).